src/ui/public/js/govukFrontend.js.LICENSE.txt

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Not sure if we should keep this in our repo? It's auto generated since the latest version.

Reply:
shall we remove it and see if everything works